### ISSUE TITLE
pg_stat_statements not working

### ISSUE THREAD

**Issue by ketys** [zalando/PGObserver#104]
Hi, I've got a problem with logging pg_stat_statements ... I've got the module enabled in database (view pg_stat_statements exists in Schemas->public->Views) and there's of course some data. But when I try to access it on frontend, it doesn't show any statistics ... Also host selection is empty (see the picture).

<img width="382" alt="screen shot 2017-04-28 at 11 28 52" src="https://cloud.githubusercontent.com/assets/6716554/25523145/ed978800-2c05-11e7-8655-4bb24f06a83f.png">

Error log writes only this: `ERROR: function zz_utils.get_load_average() does not exist`, but I think it's not related to my problem. Any help would be appreciated :)

**Comment by ketys** [comment]
Ah, maybe I got it ... statStatementsInterval should not be 0? :)

**Comment by LappleApple** [comment]
@slitsche @a1exsh Can you respond to @ketys?

**Comment by ketys** [comment]
Yes, I solved it by setting statStatementsInterval to non-zero number :) Thanks (maybe consider to mention this in documentation :))

**Comment by LappleApple** [comment]
@ketys would you mind making a PR to add it in the docs? we'll merge it then. :)

**Comment by dholleran1** [comment]
Hi, I'm also having a problem with this. Where is the statStatementsInterval setting you mention?

**Comment by a1exsh** [comment]
@dholleran1 please go to `/hosts` URI, there you should see a Settings column which defines a JSON object with settings for gathering data for the hosts.  Add `"statStatementsInterval": 600`, for example if not already there.

**Comment by dholleran1** [comment]
Yeah I found it. Thanks!
